dev-tools/reconfigurator-sp-updater/src/main.rs

```diff
@@ -374,8 +377,40 @@ struct SetArgs {
 #[derive(Clone, Debug, Subcommand)]
 enum Component {
     Sp {
+        /// expected version of the active slot
+        #[arg(long, short = 'a')]
```

Comment:
take it or leave it: I don't like short options and I would remove them.

Reply:
😄 hahaha, I just find them so useful when I'm running the commands several times! I've left them there, but if you feel really strongly about this, happy to remove them in a follow up PR